applanix_bridge/launch/applanix_bridge.launch

```diff
@@ -3,7 +3,8 @@
   <arg name="publish_tf" default="True"/>
   <arg name="odom_frame" default="gps_odom"/>
   <arg name="base_frame" default="base_link"/>
-  <arg name="zero_start" default="True"/>
+  <arg name="zero_start" default="False"/>
+  <arg name="origin" default="None"/>
```

Comment:
The default needs to be none, but I can add notes to the ROS wiki on how to set that.